docs/custom_build.md

```diff
@@ -20,9 +34,56 @@ custom_build(
 )
 ```
 
-One piece of this needs more explanation: `$EXPECTED_REF`.
+Tilt will run this command to build the image, verify that the image is in Docker,
+then push the image to the appropriate image registry.
+
+You can also use this to use `docker` flags that the `docker_build()` function doesn't support.
```

Comment:
"...also use this pattern..." OR "...also invoke `docker build` via `custom_build` to use flags that..."

Reply:
done